dags/xcom_gcs_ds.py

```diff
@@ -3,25 +3,19 @@
 from airflow.utils.dates import days_ago
 
 from datetime import datetime
-import json
-import lightgbm
-# import math
-# import matplotlib.pyplot as plt
-# import seaborn as sns
 import pandas as pd
 import numpy as np
-from sklearn.model_selection import train_test_split, RepeatedStratifiedKFold
-from sklearn.metrics import recall_score, auc, accuracy_score, roc_auc_score, roc_curve
-from sklearn.ensemble import RandomForestClassifier, GradientBoostingClassifier
-from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
-from sklearn.metrics import accuracy_score
+from sklearn.model_selection import RepeatedStratifiedKFold
 from sklearn.model_selection import cross_val_score
-from sklearn.metrics import classification_report, confusion_matrix
 
 from lightgbm import LGBMClassifier
 
-from google.cloud import bigquery
+from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook
 
+docs = """
+By default, Airflow stores all return values in XCom. However, this can introduce complexity, as users then have to consider the size of data they are returning. Futhermore, since xcoms are stored in the Airflow database by default, intermediary data is not easily accessible by external systems.
+By using an external XCom backend, users can easily push and pull all intermediary data generated in their DAG in GCS.
+"""
 
 @dag(
     default_args={'owner': 'airflow'},
```

Comment:
FYI - There is an ongoing effort to clean up how `default_args` are used in example DAGs for OSS but here, this value is the default. This can be removed.

Let's make the `start_date` value a static date rather than a dynamic one.  This is being cleaned up in example DAGs as well.